Content.Shared/GameObjects/Components/Pulling/SharedPullableComponent.cs

```diff
@@ -67,12 +67,12 @@ public bool CanStartPull(IEntity puller)
                 return false;
             }
 
-            if (!puller.TryGetComponent(out IPhysicsComponent? physics))
+            if (!Owner.TryGetComponent(out IPhysicsComponent? ownerPhysics))
```

Comment:
If Physics is a component componentdependency... then how come we also get the owner's physics component here?

Reply:
Fixed there and in a couple other places.